src/Block/BannerBlock.php

```diff
@@ -47,6 +47,19 @@ public function getCMSFields()
         return parent::getCMSFields();
     }
 
+    /**
+     * For the backend, e.g. $block->CallToActionLink, return the database column which is
+     * a json encoded string.  If there is old data where null was mistakenly saved as a string 'null'
+     * then convert that to null
```

Comment:
I would put this comment in the body of the method. Developers calling this method from another class don't need to understand the underlying purpose.

```suggestion
     * Accessor for `CallToActionLink`.
```

Reply:
Good suggestion, have updated comments